<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Push Sender</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>310</width>
     <height>370</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbStatus">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>261</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>JSON:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="teJSON"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formAppleCertificate">
     <item row="0" column="0">
      <widget class="QLabel" name="lbAppleCertificate">
       <property name="text">
        <string>Apple Certificate:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="leP12FileLocation">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>p12 file location</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="lbId">
       <property name="text">
        <string>Device ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="leId"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>231</width>
      <height>49</height>
     </rect>
    </property>
    <property name="title">
     <string>OS Type</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbIos">
        <property name="text">
         <string>iOS</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAndroid">
        <property name="text">
         <string>Android</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>horizontalLayoutWidget_3</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>verticalLayoutWidget</zorder>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="lbP12FilePassword">
       <property name="text">
        <string>p12 File Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="leP12FilePassword"/>
     </item>
    </layout>
   </widget>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>formLayoutWidget_2</zorder>
   <zorder>formLayoutWidget_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>formLayoutWidget_4</zorder>
   <zorder>lbStatus</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
